tests/nlu/training_data/test_training_data.py

```diff
@@ -122,12 +122,12 @@ def test_demo_data(files):
     td = training_data_from_paths(files, language="en")
```

Comment:
do we have a test for the new validation of `responses` in `TrainingData`?

Reply:
isn't that what https://github.com/RasaHQ/rasa/pull/6323/files#diff-91cfb497152a74c0719d07aaae3db53dR304 does? there is nothing more to test apart from the key being present and not empty, the validation isn't more specific.